Docs/source/usage/parameters.rst

```diff
@@ -432,13 +432,16 @@ Setting up the field mesh
 
 * ``geometry.dims`` (`string`)
     The dimensions of the simulation geometry.
-    Supported values are ``1``, ``2``, ``3``, ``RZ``.
-    For ``3``, a cartesian geometry of ``x``, ``y``, ``z`` is modeled.
-    For ``2``, the axes are ``x`` and ``z`` and all physics in ``y`` is assumed to be translation symmetric.
-    For ``1``, the only axis is ``z`` and the dimensions ``x`` and ``y`` are translation symmetric.
-    For ``RZ``, we apply an azimuthal mode decomposition, with ``warpx.n_rz_azimuthal_modes`` providing further control.
+    Supported values are ``1``, ``2``, ``3``, ``RZ``, ``RCYLINDER``, and ``RSPHERE``.
 
-    Note that this value has to match the :ref:`WarpX_DIMS <building-cmake-options>` compile-time option.
+    * For ``3``, a cartesian geometry of ``x``, ``y``, ``z`` is modeled.
+    * For ``2``, the axes are ``x`` and ``z`` and all physics in ``y`` is assumed to be translation symmetric.
+    * For ``1``, the only axis is ``z`` and the dimensions ``x`` and ``y`` are translation symmetric.
+    * For ``RZ``, we apply an azimuthal mode decomposition, with ``warpx.n_rz_azimuthal_modes`` providing further control.
+    * For ``RCYLINDER``, one dimensional cylinder, radius
+    * For ``RSPHERE``, one dimensional sphere, radius
```

Comment:
Should we say something more verbose like in the entries above?
```suggestion
    * For ``RCYLINDER``, the only axis is `r` and all physics is invariant under rotations in `theta` and translations in `z`.
    * For ``RSPHERE``, the only axis is `r` and all physics is invariant under rotations in `theta` and `phi`.
```
Not sure if this suggestion is the most accurate way of describing the two geometries.

Reply:
@EZoni I've updated this. To be consistent, I changed the description for all geometries. How is this?